src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const objCalc = {
+    result: 0,
+
+    operate(callback, num) {
+      callback.call(this, num);
+
+      return this;
+    },
+
+    add(num = 0) {
+      this.result += num;
+
+      return this;
```

Comment:
By the condition of the assignment, only the `operate` and `reset` functions return an object, to create a chain of calls